modelling/src/main/java/org/axonframework/modelling/command/inspection/AggregateModel.java

```diff
@@ -39,4 +41,24 @@
      * @return The current version of the aggregate
      */
     Long getVersion(T target);
+
+    /**
+     * Gets the aggregate class based on given {@code declaredType}.
+     *
+     * @param declaredType the declared type of aggregate represented by {@link String}
+     * @return the concrete aggregate class based on {@code declaredType}, if exists
+     */
+    default Optional<Class<?>> type(String declaredType) {
+        return Optional.empty();
+    }
+
+    /**
+     * Gets the declared type based on given {@code type}.
```

Comment:
Multiple usages of the term "type". It would help to make it clear that one of the "aggregate type" as used in the DomainEventMessage, while the other is the actual class of the aggregate.